.github/workflows/macos11.yml

```diff
@@ -0,0 +1,46 @@
+run-name: macOS-11_${{ vars.current_date }}_unstable.${{ github.run_id }}.${{ github.run_attempt }}
+on:
+  workflow_dispatch:
+    inputs:
+      CUSTOM_REPOSITORY:
+        description: 'Custom repository (owner/repo)'
+        required: false
+      CUSTOM_REPOSITORY_HASH:
+        description: 'Commit hash'
+        required: false
+  pull_request_target:
+    types: labeled
+    paths:
+    - 'images/macos/**'
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  update_vars:
+    if: ${{ (github.event.label.name == 'macos-all') || (github.event.label.name == 'macos-11') || (inputs.CUSTOM_REPOSITORY != '' && inputs.CUSTOM_REPOSITORY_HASH != '') || (github.event_name == 'schedule') }}
+    name: Update Variables
+    runs-on: windows-latest
```

Comment:
Any reason to run this job on `windows-latest`? can we run it on `ubuntu-latest` instead? Assignment time for ubuntu is shorter and the number of free ubuntu vms is bigger too.

Reply:
Removed (belong to the removed job)